icu4c/source/i18n/unicode/ucoleitr.h

```diff
@@ -102,6 +102,10 @@ typedef struct UCollationElements UCollationElements;
 /**
  * Open the collation elements for a string.
  *
+ * The UCollationElements retains a pointer to the supplied text.
```

Comment:
Misplaced I think. Please move this to ucol_openElements(). We normally document the text lifetime on functions that take text inputs.

Reply:
Hmm... I think maybe the diff on GitHub might be confused or perhaps it doesn't show the full comment block(?).
If I expand the diff view on GitHub it looks like this is on the API `ucol_openElements`...

I think if I fix up some of the other changes that you suggested (thanks!), then maybe the diff might be larger, and thus easier to view.

Reply:
Sorry! I must have been confused by what was shown vs. hidden.